Binary files orig/SCArray/src/.sc_misc.cpp.swp and patched/SCArray/src/.sc_misc.cpp.swp differ
diff -Nru orig/SCArray/src/sc_misc.cpp patched/SCArray/src/sc_misc.cpp
--- orig/SCArray/src/sc_misc.cpp	2023-10-20 20:46:28.000000000 +0200
+++ patched/SCArray/src/sc_misc.cpp	2024-03-16 11:39:37.406491200 +0100
@@ -17,9 +17,9 @@
 // You should have received a copy of the GNU General Public License
 // along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-#include "sc_main.h"
 #include <cstdint>
 #include <vector>
+#include "sc_main.h"
 
 
 extern "C"
